websites/O/Otakudesu/presence.ts

```diff
@@ -0,0 +1,49 @@
+const presence = new Presence({
+    clientId: "794916348761210920"
+}),
+presenceData: PresenceData = {
+    largeImageKey: "logo"
+};
+
+presence.on("UpdateData", async () => {
+  presenceData.startTimestamp = Math.floor(Date.now() / 1000);
```

Comment:
same as other pr :)

Reply:
Solved, why did you misuse this feature?

Reply:
Look, if you are going to be his friend and laywer, cool, but he can talk right? I'm not going to respond to this because you are not pull request author. 😀

Reply:
I don't know him